src/css_lexer.mll

```diff
@@ -32,6 +32,7 @@ let tailwindcss_dot = "\\."
 let tailwindcss_frac = "\\/"
 let tailwindcss_l_paren = "\\("
 let tailwindcss_r_paren = "\\)"
+let percentage = "\\%"
```

Comment:
please add prefix `tailwindcss_*`, which means the percent of tailwindcss class.